pkg/services/ngalert/sender/router.go

```diff
@@ -193,12 +192,12 @@ func (d *AlertsRouter) SyncAndApplyConfigFromDatabase() error {
 
 	// We can now stop these externalAlertmanagers w/o having to hold a lock.
 	for orgID, s := range sendersToStop {
-		d.logger.Info("stopping sender", "org", orgID)
+		d.logger.Info("Stopping sender", "org", orgID)
 		s.Stop()
-		d.logger.Info("stopped sender", "org", orgID)
+		d.logger.Info("Stopped sender", "org", orgID)
 	}
 
-	d.logger.Debug("finish of admin configuration sync")
```

Comment:
I would keep it debug because this is the hot path and info can be noisy

Reply:
For some reason I was under the impression that this only ran in the background _if we detected that there was a change in the admin config_. So, this runs much more often than I originally thought - a lot can be tuned down knowing this.